docs/getting-started/getting-started.md

```diff
@@ -6,13 +6,13 @@ sidebar_label: Getting Started
 
 ## Installation & Download
 
-The Radicle network can be accessed via Upstream, a desktop client. To download the app, click [here](https://buildkite.com/monadic/radicle-upstream/builds/6098). While Upstream is currently the only client for using Radicle, it won't be the only one. Upstream development is public by default and completely open source. You can follow along development on [radicle.community](https://radicle.community)
+The Radicle network can be installed and accessed via Upstream, a desktop client. To download the app, click [here](https://buildkite.com/monadic/radicle-upstream/builds/6098). While Upstream is currently the only client for using Radicle, it won't be the only one. Upstream development is public by default and completely open source. You can follow along development on [radicle.community](https://radicle.community)
```

Comment:
This sounds a bit like we're installing the radicle network on our machine, which to me doesn't make sense. What was wrong with the previous version?

Reply:
Was trying to reflect the concept of "Radicle installation" that @xla brought up, but you're right, doesn't make sense with the subject. Will revert back.